docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: '3.8'
+
+services:
+  db:
+    build:
+      context: .
+      dockerfile: Data/Dockerfile
+    environment: 
+      - MSSQL_SA_PASSWORD=Dexcelence!1
+    networks:
+      mssql-network:
+        ipv4_address: 172.16.238.2
+
+  identity:
+    build:
+      context: .
+      dockerfile: IdentityServer/Dockerfile.dev
+    environment: 
+      - App__Self__IssuerUri=http://172.16.238.3:5004
+      - App__Api__DeXApiUrl=http://172.16.238.4
+      - ASPNETCORE_URLS=https://+;http://+
+      - ASPNETCORE_HTTPS_PORT=5005
+      - ASPNETCORE_Kestrel__Certificates__Default__Password=W64x4AD8dNj9kImdX3tayS
+      - ASPNETCORE_Kestrel__Certificates__Default__Path=/app/dex-identity.pfx
+    networks: 
+      mssql-network:
+        ipv4_address: 172.16.238.3
+    ports:
+      - 5004:80
+      - 5005:443
+    
+  api:
+    build:
+      context: .
+      dockerfile: API/Dockerfile.dev
+    depends_on:
+      - db
+    environment: 
+      - ConnectionStrings__DefaultConnection=Server=172.16.238.2;Database=master;User=sa;Password=Dexcelence!1;
```

Comment:
We've removed all the secrets from the code base and now you add it in the docker-compose?

Reply:
docker-compose.prod.yml has been added. This file is for development purposes only.